src/Components/test/E2ETestMigration/Microsoft.AspNetCore.Components.Migration.E2ETests.csproj

```diff
@@ -12,7 +12,8 @@
     <TestTrimmedApps Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</TestTrimmedApps>
     <TestTrimmedApps Condition="'$(Configuration)' == 'Release'">true</TestTrimmedApps>
 
-    <!-- Skip the Components E2ETestMigration in CI -->
+    <!-- Skip building and running the Components E2ETestMigration in CI -->
+    <ExcludeFromBuild Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</ExcludeFromBuild>
```

Comment:
Should we leave this building? I worry we wouldn't find out if this stops compiling if we turn off building on the CI. What would be preferable is to remove the `TestTrimmedApps` condition so we don't cause  trimming to happen as part of building this project. e.g.

```diff
- <TestTrimmedApps Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</TestTrimmedApps>
- <TestTrimmedApps Condition="'$(Configuration)' == 'Release'">true</TestTrimmedApps>

- <ExcludeFromBuild Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</ExcludeFromBuild>
```